docs/swagger.yml

```diff
@@ -1289,6 +1306,8 @@ definitions:
         description: "The maximum concurrent host scans. default is 50."
       disabled_optional_products:
         $ref: "#/definitions/DisableOptionalProducts"
+      enable_extended_product_search:
```

Comment:
`enable_extended_product_search` --> `enabled_extended_product_search`
Need to keep consistent with `disabled_optional_products`

Reply:
Done.